web/src/beta/features/PluginPlayground/hooks.tsx

```diff
@@ -29,19 +30,47 @@ export default () => {
     sharedPlugin
   } = usePlugins();
 
-  const { widgets, executeCode } = useCode({
+  const { infoboxBlocks, widgets, executeCode } = useCode({
     files: selectedPlugin.files
   });
 
-  const [layers, setLayers] = useState<Layer[]>(
-    DEFAULT_LAYERS_PLUGIN_PLAYGROUND
-  );
+  const [selectedLayerId, setSelectedLayerId] = useState("");
+  const [visibleLayerIds, setVisibleLayerIds] = useState<string[]>(["1", "2"]); // Note: the layerIds in web/src/beta/features/PluginPlayground/LayerList/constants.ts
```

Comment:
```suggestion
  const [visibleLayerIds, setVisibleLayerIds] = useState<string[]>(DEFAULT_LAYERS_PLUGIN_PLAYGROUND.map((l) => l.id)); 
```
We can use the ids from data instead of hard code.

Reply:
Thanks. This has been refactored.